src/org/etools/j1939_84/bus/j1939/J1939.java

```diff
@@ -667,7 +656,7 @@ private ParsedPacket processRaw(int pgn, Packet packet) {
                                                                    boolean fullString,
                                                                    Class<T> clas,
                                                                    Packet requestPacket) {
-        return requestGlobal(title, listener, fullString, clas, null, requestPacket);
+        return requestGlobal(title, listener, fullString, clas, getPgn(clas), requestPacket);
```

Comment:
Step 26 needs to be able to request any PGN and the result (`clas`) is GenericPacket which doesn’t have a PGN field..  This needs reverted

Reply:
How can step 26 request a pgn without a pgn?  I think that GenericPacket not having a PGN is independent of this API.  We are never requesting GenericPacket.